templates/download/desktop/index.html

```diff
@@ -34,11 +34,14 @@ <h2>Ubuntu {{ releases.lts.full_version }} LTS</h2>
       </div>
     </div>
   </div>
+
+  {% if releases.latest.short_version > releases.lts.short_version %}
```

Comment:
I think you mean the opposite. `lts > latest`. Currently, this displays 19.10 on the download page.

Reply:
is this because the releases.yaml is out of sync? what if you increase the short form of the lts to 20.04?